src/generator.rs

```diff
@@ -311,6 +313,66 @@ const ENUM: &str = r#"<name> = {
     end,
 }"#;
 
+const ANY_METHOD_SIGNATURES: &str = r#"
+-- Pack a message into an Any.
+--
+-- If typePrefix is not provided, defaults to "apis.roblox.com/cloud".
+pack: <T, F>(payload: proto.Message<T, F>, typePrefix: string?) -> Any,
+
+-- Returns the message contained by the Any (or nil if the Any is empty).
+unpack: <T, F>(self: Any, registry: typeRegistry.TypeRegistry?) -> proto.Message<T, F>?,
+
+-- Returns true iff the Any contains an object of the type specified by
+-- typeName. If typeName is a full type URL, it will be compared; otherwise,
+-- only the type name will be compared.
+isA: (self: Any, typeName: string) -> boolean,
+"#;
+
+const ANY_METHODS: &str = r#"
```

Comment:
Can we put this in its own luau file (that isn't luau analyzed)? I think the generator.rs is pretty full on Lua prose